src/ascend.ts

```diff
@@ -146,14 +149,70 @@ type MoonSign =
   | "gnomish"
   | "gnomish gnomads camp";
 
+/**
+ * @param moon Moon sign name
+ * @returns Moon sign id else 0
+ */
+export function signNameToId(moon: MoonSign) {
+  switch (moon.toLowerCase()) {
+    case "mongoose":
+      return 1;
+    case "wallaby":
+      return 2;
+    case "vole":
+      return 3;
+    case "platypus":
+      return 4;
+    case "opossum":
+      return 5;
+    case "marmot":
+      return 6;
+    case "wombat":
+      return 7;
+    case "blender":
+      return 8;
+    case "packrat":
+      return 9;
+  }
+  return 0;
+}
+
+/**
+ * @param id Moon sign id
+ * @returns Name of moon sign else "None"
+ */
+export function signIdToName(id: number) {
+  switch (id) {
+    case 1:
+      return "Mongoose";
+    case 2:
+      return "Wallaby";
+    case 3:
+      return "Vole";
+    case 4:
+      return "Platypus";
+    case 5:
+      return "Opossum";
+    case 6:
+      return "Marmot";
+    case 7:
+      return "Wombat";
+    case 8:
+      return "Blender";
+    case 9:
+      return "Packrat";
+  }
+  return "None";
+}
```

Comment:
These feel like they should just use objects to store the information instead of big switch statements

also the inconsistent casing is a little odd

Reply:
Eh maybe. I'd have to do two equally sizes objects no?

Reply:
You could use a single array; for `signIdToname` you'd do `moonSigns[id - 1] ?? "None"` and for `signNameToId` you'd do `moonSigns.indexOf(moon) + 1`

Reply:
Yeah I suppose that search doesn't really cost much in the grand scheme of things